loopchain/peer/block_manager.py

```diff
@@ -676,10 +674,13 @@ def stop(self):
         if self.consensus_algorithm:
             self.consensus_algorithm.stop()
 
+    def new_epoch(self):
+        self.epoch = Epoch.new_epoch()
+
     def leader_complain(self):
-        complained_leader_id = self.epoch.leader_id
+        complained_leader_id = self.epoch.prev_leader_id or self.epoch.leader_id
```

Comment:
What does it become complained leader id when multiple times complained?
For example:
- Leader order : hxaa, hxbb, hxcc

block1(hxaa), complained
-> block1(hxbb, complained_leader_id: hxaa), complained 
-> block1(hxcc, complained_leader_id: hxbb)
Is it right? 

Or
block1(hxaa), complained
-> block1(hxbb, complained_leader_id: hxaa), complained 
-> block1(hxcc, complained_leader_id: `hxaa`)

Reply:
prev_leader_id in epoch stand for last complained leader in same epoch.